addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -287,46 +287,26 @@
-                            <div class="oe_kanban_global_click o_kanban_record_has_image_fill px-0 pb-0">
-                                <div class="oe_kanban_details d-flex flex-column">
-                                    <div class="o_kanban_record_top px-2 h-50">
-                                        <img t-att-src="kanban_image('hr.employee', 'avatar_128', record.employee_id.raw_value)" t-att-title="record.employee_id.value" t-att-alt="record.employee_id.value" class="o_image_40_cover float-start"/>
-                                        <div class="o_kanban_record_headings ps-2 pe-2">
-                                            <div class="text-truncate">
-                                                <strong class="o_kanban_record_title">
-                                                    <span t-esc="record.project_id.value" t-att-title="record.project_id.value"/>
-                                                </strong>
-                                            </div>
-                                            <div class="text-truncate">
-                                                <i><span t-esc="record.task_id.value" t-att-title="record.task_id.value"/></i>
-                                            </div>
-                                            <div class="text-truncate">
-                                                <span t-esc="record.name.value" t-att-title="record.name.value"/>
-                                            </div>
-                                        </div>
-                                    </div>
-                                    <div class="o_kanban_record_bottom d-block mx-2">
-                                        <hr class="mt4 mb4"/>
-                                        <span>
-                                            <i class="fa fa-calendar me-1" role="img" aria-label="Date" title="Date"></i>
-                                            <t t-esc="record.date.value"/>
-                                        </span>
-                                        <span class="float-end" name="duration">
-                                            <strong>Duration: </strong><field name="unit_amount" widget="timesheet_uom" decoration-danger="unit_amount &gt; 24" decoration-muted="unit_amount == 0"/>
-                                        </span>
-                                    </div>
+                        <t t-name="kanban-card">
+                            <div class="d-flex flex-row">
+                                <field name="employee_id" widget="image" options="{'preview_image': 'avatar_128'}" t-att-title="record.employee_id.value" t-att-alt="record.employee_id.value" class="o_image_40_cover me-2 float-start"/>
+                                <div class=" d-flex flex-column text-truncate lh-sm">
+                                <!-- TODO: Previously, we could see the preview of the below field because t-esc was used.
```

Comment:
We'll have to find a way to make this work. In the meantime, you can wrap those field nodes inside a span (or strong) on which you can set the t-att-title